### PR TITLE
584 handle sony ci webhook

### PR DESCRIPTION
**Pull request by afred**

### CODE REVIEW

**Review by foglabs — CHANGES_REQUESTED** [review]
I think the things I tagged are a copy paste error, but otherwise looks 👍

File: app/controllers/sony_ci/webhook_logs_controller.rb [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    end
+  end
+
+  # PATCH/PUT /sony_ci/webhook_logs/1
+  # PATCH/PUT /sony_ci/webhook_logs/1.json
+  def update
+    respond_to do |format|
+      if @sony_ci_webhook_log.update(sony_ci_webhook_log_params)
+        format.html { redirect_to @sony_ci_webhook_log, notice: 'Webhook log was successfully updated.' }
+        format.json { render :show, status: :ok, location: @sony_ci_webhook_log }
```
[the remark this answers is not in the document]
Comment (afred) [reply app/controllers/sony_ci/webhook_logs_controller.rb:46]: Ack... i meant to get rid of all actions pertaining to WebhookLogs that were not `#show` or `#index`. The `WebhgookLog` records get created by virtue of Sony Ci sending requests to the `/webhooks/[action]` endpoint(s), so we don't currently have a use case for adding/editing/deleting the `WebhookLog` records directly. I'll fix this.

fwiw, you're correct, I think those actions responding with `format.json` will try to render a `app/views/sony_ci/webhook_logs/[action].json.jbuilder`, which is a ruby file that uses a ruby DSL to build JSON structures from `ActiveRecord` models. I believe I removed the tests for these controller actions, hence no errors on Travis.

**Review by foglabs — APPROVED** [review]

**Comment by afred** [comment]
NOTE: the branch name beginning in `585` is misleading here, this PR is a solution for #584, not 585.

**Comment by afred** [comment]
and finally, in lieu of Travis buids...
image

**Pull request merged**